lib/upipe-dvbcsa/upipe_dvbcsa_encrypt.c

```diff
@@ -292,10 +571,6 @@ static struct upipe_mgr upipe_dvbcsa_enc_mgr = {
     .upipe_control = upipe_dvbcsa_enc_control,
 };
 
-/** @This returns the dvbcsa encryption pipe management structure.
- *
- * @return a pointer to the management structure
- */
 struct upipe_mgr *upipe_dvbcsa_enc_mgr_alloc(void)
```

Comment:
Why did you remove the doc ?

Reply:
https://github.com/Upipe/upipe/blob/master/lib/upipe-dvbcsa/upipe_dvbcsa_bs_encrypt.c#L514 I did not

Reply:
You removed it from upipe_dvbcsa_encrypt =)

Reply:
No I used git mv ..